evap/static/css/evap.css

```diff
@@ -511,10 +511,19 @@ li.menu-indent {
 }
 
 .text-answer li,a {
-    word-break: break-word;
+    word-wrap: break-word;
```

Comment:
What do you mean resolved? break-word is a property of word-wrap and not word-break. The way it is right now it shouldn't work:

http://www.w3schools.com/cssref/css3_pr_word-wrap.asp

http://www.w3schools.com/cssref/css3_pr_word-break.asp